central/complianceoperator/v2/report/request.go

```diff
@@ -16,5 +16,25 @@ type Request struct {
 	Ctx                context.Context
 	SnapshotID         string
 	NotificationMethod storage.ComplianceOperatorReportStatus_NotificationMethod
-	FailedClusters     map[string]*storage.ComplianceOperatorReportSnapshotV2_FailedCluster
+	ClusterData        map[string]*ClusterData
+	FailedClusters     int
```

Comment:
Oh, here it is! Let's rename it
```suggestion
	NumFailedClusters     int
```

Reply:
Done